### DISCUSSION TITLE
Framework para Newsletter

### DISCUSSION THREAD

**Discussion by juao** [frontendbr/forum#2076] [Discussão]
Fala Fórum,

Me surgiu uma dúvida sobre criação de e-mails transacionais, newsletter etc dos sistemas que montamos no dia a dia.

Vocês fazem tudo na mão ou utilizam algum framework como o Foundation, MJML, Maizzle, etc para auxiliar na criação no dia a dia?

Pergunto mais pelo motivo de hoje os e-mails precisarem ser responsivo e também funcionar corretamente em diversos leitores de e-mails, gostaria de dar uma padronizada neles e também agilizar o tempo de testes adotando alguém framework. 

Vocês indicam algum? Como é a experiência de vocês nessa tipo de criação?

Abraço a todos.

**Comment by glauberm** [comment]
Trabalhei com isso há um tempo e utilizei o [HEML](https://heml.io/). O Foundation na época me pareceu abandonado e o MJML um pouco inchado para o que eu queria. Minhas necessidades eram simples, basicamente só um template de newsletter com alguns blocos pré-definidos que variavam. Então eu só utilizei o CLI do HEML para gerar o HTML desses templates e desses blocos, e integrei esse HTML ao projeto.

**Reply by juao** [reply]
Obrigado pelo feedback amigo!

**Comment by ibrahimcesar** [comment]
Utilizamos o [MJML](https://mjml.io/). Ele tem a facilidade de poder se utilizado localmente apenas ou como uma biblioteca em uma Lambda por exemplo. Testamos de tudo, foi o que melhor se adaptou a nosso time, em um contexto em que trabalhamos com Node regularmente. Geralmente o utilizamos apenas na fase de release de um novo layout – depois os editores utilizam a versão editável na ferramenta de newsletter. Nós utilizamos no contexto de emails transacionais, emails de marketing e newsletters informativas.
